manifests/charts/base/templates/services.yaml

```diff
@@ -24,5 +24,11 @@ spec:
   type: ExternalName
   externalName: {{ .Values.global.remotePilotAddress }}
   {{- end }}
+  {{- if .Values.global.ipFamilyPolicy }}
```

Comment:
where do we use global.ipFamilyPolicy? Did not see that in the example you have used.